models/Session.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+let schema = new Schema({
+    type: {type: String, required: true}, //"type = {"Login", "Logout"}
+    _id: {type: String, required: true},
```

Comment:
We don't want to use `_id` here. `_id` is a keyword in mongodb and it represents a primary key. By making an `_id` field you're not allowing the db to store multiple session objects from the same user. In fact, take a look at the mlab right now and you'll see there is only 1 session document.